app/assets/stylesheets/content/_tables.sass

```diff
@@ -58,6 +58,40 @@ table
     .hours-dec
       font-size: 0.9em
 
+  &[class*=transitions]
```

Comment:
I could imagine that this selector will match more often than anticipated. and you only want to match the three tables on the workflow page.

Can't you either specify all three classes explicitly or add another (common) class to all of them? The current selector will also match for `-smooth-transitions` etc.